packages/sdk/README.md

```diff
@@ -547,24 +547,93 @@ Also see [TeamsFx class](#teamsfx-class) for furthur description.
 
 From `botbuilder@4.16.0`, `BotFrameworkAdapter` is deprecated, and `CloudAdapter` is recommended to be used instead. You can import `ConversationBot` from `BotBuilderCloudAdapter` to use the latest SDK implemented with `CloudAdapter`.
 
-```ts
-const { BotBuilderCloudAdapter } = require("@microsoft/teamsfx");
-const ConversationBot = BotBuilderCloudAdapter.ConversationBot;
-
-const commandBot = new ConversationBot({
-  // The bot id and password to create CloudAdapter.
-  // See https://aka.ms/about-bot-adapter to learn more about adapters.
-  adapterConfig: {
-    MicrosoftAppId: config.botId,
-    MicrosoftAppPassword: config.botPassword,
-    MicrosoftAppType: "MultiTenant",
-  },
-  command: {
-    enabled: true,
-    commands: [new HelloWorldCommandHandler()],
-  },
-});
-```
+1. Install `@microsoft/teamsfx @^2.2.0`, `botbuilder @^4.18.0`, (and `@types/node @^14.0.0` for TS projects) via `npm install` as follows.
+
+    ```sh
+    npm install @microsoft/teamsfx
+    npm install botbuilder
+    
+    // For TS projects only
+    npm install --save-dev @types/node
+    ```
+
+2. Update the import of `ConversationBot` and create a new `ConversationBot` as follows.
+
+    ```ts
+    const { BotBuilderCloudAdapter } = require("@microsoft/teamsfx");
```

Comment:
This is a js import style and inconsistent with L602. Better make them consistent.

Reply:
Thanks! Update the code snippet with TS code instead.